store/sqlstore/user_store.go

```diff
@@ -257,14 +257,12 @@ func (us SqlUserStore) UpdatePassword(userId, hashedPassword string) store.Store
 	})
 }
 
-func (us SqlUserStore) UpdateFailedPasswordAttempts(userId string, attempts int) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		if _, err := us.GetMaster().Exec("UPDATE Users SET FailedAttempts = :FailedAttempts WHERE Id = :UserId", map[string]interface{}{"FailedAttempts": attempts, "UserId": userId}); err != nil {
-			result.Err = model.NewAppError("SqlUserStore.UpdateFailedPasswordAttempts", "store.sql_user.update_failed_pwd_attempts.app_error", nil, "user_id="+userId, http.StatusInternalServerError)
-		} else {
-			result.Data = userId
-		}
-	})
+func (us SqlUserStore) UpdateFailedPasswordAttempts(userId string, attempts int) (string, *model.AppError) {
```

Comment:
Looks like the returned `userId` is never used, do you mind to remove it from the returns values and just return the `*model.AppError`?

Reply:
Let's improve this code!